### ISSUE TITLE
No label here for issues with Azure / docker-for-azure

### ISSUE THREAD

**Issue by nealmcb** [docker/docker.github.io#7978]
### Problem description

TL;DR: Please create an _Azure_ label for issues in this github repo.

I'm wondering how up-to-date the Azure documentation is here, and a github issue label like _Azure_ would help me find issues others have found e.g. with images of Azure portal screenshots which I can't even match up with what I'm seeing on Azure.

### Problem location

- I saw a problem on the following URL: <URL>

In https://github.com/docker/docker.github.io/issues the _labels_ dropdown shows many possible issue labels, e.g. [content/cli](https://github.com/docker/docker.github.io/issues?q=is%3Aopen+is%3Aissue+label%3Acontent%2Fcli), but none satisfying the filter "Azure".

- I couldn't find the information I wanted. I expected to find it near the following URL: <URL>

- Other: <DETAILS>

### Suggestions for a fix

Please create an _Azure_ label for issues in this github repo.
